pyproject.toml

```diff
@@ -8,9 +8,12 @@ authors = []
 python = "^3.7"
 flask = "^1.1.2"
 python-dotenv = "^0.14.0"
+requests = "^2.25.1"
+selenium = "^3.141.0"
```

Comment:
This can go in `dev-dependencies` instead

Reply:
done